hop/apps/SNalert/decider.py

```diff
@@ -3,45 +3,39 @@
 from . import IDecider
 
 class Decider(IDecider.IDecider):
-    def __init__(self, time_threshold, datetime_format, mongoServer, drop_db):
+    def __init__(self,
+                 coinc_threshold=10,
+                 msg_expiration=120,
+                 datetime_format="%y/%m/%d %H:%M:%S",
+                 mongo_server="mongodb://localhost:27017/",
+                 drop_db=True):
         """
         The constructor.
-        :param time_threshold:
-        :param datetime_format:
-        :param mongoServer:
-        :param drop_db:
+        :param coinc_threshold: maximum time (s) between messages for them to be considered coincident
+        :param msg_expiration: maximum time (s) for a message to be stored in the database cache
+        :param datetime_format: date format to convert from a string
+        :param mongo_server: URL string of the mongodb server address
+        :param drop_db: boolean specifying whether to clear previous database storage
         """
         # intialize and use redis storage
-        self.db = db_storage.storage(time_threshold, datetime_format, mongoServer, drop_db)
-        self.tightThreshold = 10
-        self.looseThreshold = time_threshold
+        self.db = db_storage.storage(msg_expiration, datetime_format, mongo_server, drop_db)
+        self.coinc_threshold = coinc_threshold
 
     def deciding(self):
         """
 
         :return: True or false indicating the possibility of a supernova
         """
-        # if not self.db.cacheEmpty():
-        #     cacheMsgs = self.db.getCacheMsgs()
-        #     # go through messages to check if any two or more are within 10s
-        #         # if yes
-        #             # verify locations different (as long as at least two are in different locations)
-        #             # if so
-        #                 return true
-        #             # if not
-        #                 return false
-        #
-        #         # if not, no-op or print a message
-        #
```

Comment:
Some of this looks to be a flow diagram on how coincidences are detected so it might be good to incorporate that somehow into a comment/docstring if it's not already in the docs.

Reply:
Good call; I added more to the docs and docstring reflecting this and other changes.